packages/labs/package.json

```diff
@@ -9,7 +9,9 @@
   "dependencies": {
     "@blueprintjs/core": "^1.23.0",
     "classnames": "^2.2",
-    "pure-render-decorator": "1.2.1",
+    "popper.js": "1.11.0",
```

Comment:
locking down version cuz 1.11.1 has types that are incompatible with `react-popper` (cuz someone came in and rewrote them after I did and react-popper never got updated accordingly) https://github.com/FezVrasta/popper.js/releases/tag/v1.11.1